### PR TITLE
Improvement: Added Box Shadow options to Advanced columns and Section.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
Improvement: Added Box Shadow options to Advanced columns and Section.

### CODE REVIEW

File: classes/class-uagb-config.php [hunk 1]
```diff
@@ -1520,6 +1526,12 @@ public static function get_block_attributes() {
 							'gradientOverlayLocation2' => '100',
 							'gradientOverlayAngle'     => '0',
 							'gradientOverlayPosition'  => 'center center',
+							'boxShadowColor' => '#000000',
```
Comment (vrundakansara) [inline classes/class-uagb-config.php:1529]: Remove this default color.

File: classes/class-uagb-config.php [hunk 2]
```diff
@@ -162,6 +162,12 @@ public static function get_block_attributes() {
 							'mobilePaddingType'      => 'px',
 							'tabletPaddingType'      => 'px',
 							'desktopPaddingType'     => 'px',
+							'boxShadowColor' => '#000000',
```
Comment (vrundakansara) [inline classes/class-uagb-config.php:165]: Remove this default color.

Comment (RajkiranBagal) [reply classes/class-uagb-config.php]: Removed.

File: classes/class-uagb-config.php (hunk 1 above)
Comment (RajkiranBagal) [reply classes/class-uagb-config.php]: Removed.

**Comment by vrundakansara** [comment]
The box shadow is applied to inner wrap. This creates an issue for Full Width column layout.
https://share.getcloudapp.com/nOuNqOrK

Apply to the `wp-block-uagb-columns` selector for Advanced Columns. Check the same for Section too.

**Pull request merged**